hazelcast/src/test/java/com/hazelcast/client/config/XmlYamlClientConfigBuilderEqualsTest.java

```diff
@@ -116,8 +115,10 @@ public void testFullClientFailoverConfig() throws Exception {
     }
 
     private String readResourceToString(String resource) throws IOException {
-        InputStream xmlInputStream = getClass().getClassLoader().getResourceAsStream(resource);
-        return new String(IOUtil.toByteArray(xmlInputStream));
+        try (InputStream xmlInputStream = getClass().getClassLoader().getResourceAsStream(resource)) {
```

Comment:
You could replace this with [this](https://stackoverflow.com/a/6068228) and avoid a `byte[]` entirely.

Reply:
I think you are suggesting using Guava. I don't think we need an extra dependency for this. We are trying to use a little extra dependency as possible. Thanks for the suggestion anyway 👍

Reply:
> I think you are suggesting using Guava. I don't think we need an extra dependency for this. We are trying to use a little extra dependency as possible. Thanks for the suggestion anyway 👍

Yes, it’s Guava. I think its already a dependency on the project which is why I suggested it.

Reply:
Guava is not our direct dependency. It is declared under dependencyManagent. In the build system there is a maven-enforcer-plugin. It enforces all transitive dependencies but have the same version. Sometimes there is conflict and we add an entry to dependencyManagent to fix the version used.